web/src/common/constants.js

```diff
@@ -214,3 +214,10 @@ export const extendedMeasurements = Object.freeze([
 ]);
 
 export const allMeasurements = [...measurements, ...extendedMeasurements];
+
+export const constants = Object.freeze({
```

Comment:
Avoid adding constants that are only used once. Constants for single words like `divers` probably isn't useful.

Reply:
I can understand why you think the constants is use in one place only because I forgot a check in the test case that it should return data only if the url matches, I added back the if check in the test

Reply:
Removed given the test case deleted